languages/ruby/exercises/concept/booleans/attendee.rb

```diff
@@ -0,0 +1,29 @@
+class Attendee
+  attr_reader :height, :pass_id
+
+  def initialize(height)
+    @height = height
+  end
+
+  def issue_pass!(pass_id)
+    @pass_id = pass_id
+  end
+
+  def revoke_pass!
+    @pass_id = nil
+  end
+
+  # Do not edit above methods, add your own methods below.
+
+  def pass?
+    raise NotImplementedError, 'Please implement the has_pass? method'
```

Comment:
`has_pass?` or `pass?`?

Reply:
Good question.  I changed it based on the other feedback from kotp, I am inclined towards `has_pass?` as that is what I would write, but not sure about the ruby convention.

Reply:
The `is_pass?` or `has_pass?` is duplication of communication because of the question mark that Rubyists can use.  If we have `has_pass` the question mark would not be needed, it is a statement, not a question.  Similar with `is_pass`.  With `pass?` the question mark asks the question.  In this case, is the pass valid and do they even have one?  Combined in one small communicative name.

Just my thought on it.

Reply:
The Hash class would be an exception to that though, with `has_key?` rather than `key?`, no?

Reply:
Not only is there a method of that both names, it is the same method.